internal/vm/vm.go

```diff
@@ -63,6 +65,8 @@ func New(opts Opts) (*VM, error) {
 	l.PreloadModule("go.jsonschema", jsonschema.Loader)
 	l.PreloadModule("cjson.safe", safe.Loader)
 	l.PreloadModule("lyaml", lyaml.Loader)
+	l.PreloadModule("sha256", sha256.Loader)
+	l.PreloadModule("hex", hex.Loader)
```

Comment:
We're using the pattern "go.xxx" for those replacement modules:
```suggestion
	l.PreloadModule("go.sha256", sha256.Loader)
	l.PreloadModule("go.hex", hex.Loader)
```

Reply:
[33e4b3c](https://github.com/Kong/goks/pull/36/commits/33e4b3c61ffe7da93de7e518f3b37ea28611933c)